tests/test_const.py

```
@@ -0,0 +1,73 @@
+sys.modules["homeassistant.config_entries"] = MagicMock()
+sys.modules["homeassistant.core"] = MagicMock()
+sys.modules["homeassistant.const"] = MagicMock()
+sys.modules["homeassistant.helpers"] = MagicMock()
+sys.modules["homeassistant.helpers.device_registry"] = MagicMock()
+sys.modules["homeassistant.helpers.update_coordinator"] = MagicMock()
+sys.modules["homeassistant.exceptions"] = MagicMock()
+sys.modules["garminconnect"] = MagicMock()
+sys.modules["garth"] = MagicMock()
+sys.modules["garth.exc"] = MagicMock()
+
+from custom_components.garmin_connect.const import (  # noqa: E402
```

Ruff (0.14.10)

Unused `noqa` directive (non-enabled: `E402`)

Remove unused `noqa` directive

(RUF100)